### ISSUE TITLE
Playlist shortcuts are totally unusable

### ISSUE THREAD

**Issue by Ilya87** [MBach/Miam-Player#114]
Arch Linux x86_64
Shortcuts can't be used or changed - pressing C (default play/pause button) does nothing. Changing shortcuts to keyboards's multimedia play/pause can't be saved - after I close setting window and then open playlist shortcut settings again - default shortcut is set.

**Comment by MBach** [comment]
Hello,
I've made big changes when one is dealing with shortcuts :) https://github.com/MBach/Miam-Player/commit/8bfc997c3512957ddf782347fad9d3ae6133ffb5
Now if you have the menu bar hidden, shortcuts will still continue to work!
Defaults shortcuts are added into the settings, so you must need to remove your settings with this commit, or at least the entry shortcuts in General section

**Comment by Ilya87** [comment]
Custom shortcuts can't be saved. Default delete button does nothing in playlist. And what about global shortcuts?

**Comment by MBach** [comment]
Are you sure that you have deleted the file `~/.config/MmeMiamMiam/MiamPlayer.ini` ?

**Comment by Ilya87** [comment]
Yes, I deleted both config folders. As I understand I should only change shortcut and close config window? In this case after I open config window again shortcuts became default. And delete doesn't work.
